### PR TITLE
Add Blender

### PR DESCRIPTION
**Pull request by MrBrain295**
## Adding a language
Add Blender
- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

### CODE REVIEW

File: b/Blender [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Blender
+ from Blender import Scene, Text3d
```
Comment (Richienb) [inline b/Blender:2]: Are these lines really supposed to have 1 space of indent?

**Review by Richienb — APPROVED** [review]

File: b/Blender (hunk 1 above)
Comment (MrBrain295) [reply b/Blender:2]: According to the hello world collection yes

Comment (MrBrain295) [reply b/Blender:2]: I could remove them or add tabs

Comment (Richienb) [reply b/Blender:2]: Just remove the indent

**Review by Richienb — CHANGES_REQUESTED** [review]
Shouldn't this be `Blender.py`?

**Comment by MrBrain295** [comment]
Done ✅

File: readme.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Thanks to everyone who continues to contribute; new languages are created every
 Make sure to see [contributing.md](/contributing.md) for instructions on contributing to the project!
 
 <!--Languages start-->
-## Languages (756 total)
```
Comment (frasca617) [inline readme.md:11]:
```suggestion
## Languages (756 total)￼
```

**Comment by frasca617** [comment]
Duplicate of #980 #995 #994 #993

**Comment by MrBrain295** [comment]
No

**Comment by MrBrain295** [comment]
@frasca617 how is this a duplicate of those pr you mentioned

**Pull request merged**